src/CounterInterceptor/MyActivities.cs

```diff
@@ -0,0 +1,17 @@
+namespace TemporalioSamples.CounterInterceptor;
+
+using System.Diagnostics;
+using Temporalio.Activities;
+
+public class MyActivities
+{
+    [Activity]
+    public string SayHello(string name, string title) =>
+        "Hello " + title + " " + name;
```

Comment:
```suggestion
        $"Hello {title} {name}";
```

I think interpolation is a bit clearer here

Reply:
Fixed